config/routes.config.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+
+const controller = require("../controllers/spotify.controller");
```

Comment:
```suggestion
const spotifyController = require("../controllers/spotify.controller");
```

Piensa que ahora solo tenemos un controlador, pero esto es escalable, si nos acostumbramos desde ya a poner nombres a las variables que sean descriptivos vas a crecer x1000, 0 bromas :D